dev/tests/hu.elte.txtuml.validation.test/src/hu/elte/txtuml/validation/ModelTest.java

```diff
@@ -103,7 +103,7 @@ public void testInvalidModifier() throws Exception {
 
 		compilationUnit.accept(new ModelVisitor(mockCollector));
 
-		verify(mockCollector, times(3)).report(isA(InvalidModifier.class));
+		verify(mockCollector, times(2)).report(isA(InvalidModifier.class));
```

Comment:
It is explained in the (detailed) commit description:
https://github.com/ELTE-Soft/txtUML/pull/485/commits/c4d09e986a5a93b6eaa86e270eb6b5482de72329